wait-for.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
+
```

Comment:
I like the clear pseudocode to describe what's going on.  It nice to have a question answered just as you're asking it to yourself....